contrib/install_tiller.sh

```diff
@@ -7,15 +7,4 @@ kubectl -n kube-system create sa tiller \
       --clusterrole cluster-admin \
       --serviceaccount=kube-system:tiller
 
-helm init --skip-refresh --upgrade --service-account tiller
```

Comment:
Why aren't we checking for tiller to be ready?

Reply:
I added a `--wait` flag at the end of the script

Reply:
OK that resolves my concern. Can you back-port these changes to off-bootstrap too? Looks 💯